tests/helpers/equality-helpers.js

```diff
@@ -1,26 +1,31 @@
-export { rowValuesEqual, inputPropertiesEqual };
+export { rowValuesEqual, inputPropertiesEqual }
 
-function rowValuesEqual(assert, row) {
-  let values = Array.prototype.slice.call(arguments, 2, arguments.length);
-  let columns = row.find('th, td');
-  let columnText;
+function rowValuesEqual(...args) {
```

Comment:
hmmm, not quite like that but close.

`function rowValuesEqual(expect, row, ...values)` will work to name the first to and collect the remaining into an array called values.